src/Raven.Server/Commercial/LicenseStatus.cs

```diff
@@ -70,9 +70,11 @@ public bool Expired
             }
         }
 
-        public int MaxCores => GetValue<int?>("cores") ?? 3;
+        internal static bool UnderTest;
 
-        public int MaxMemory => GetValue<int?>("memory") ?? 6;
+        public int MaxCores => GetValue<int?>("cores") ?? (UnderTest ? 3 : 256);
```

Comment:
What is this?

Reply:
Should be the other way around.. (UnderTest ? 256 : 3)
If we unable to set the license we should use 256 cores as default and not 3